google/async_resumable_media/_upload.py

```diff
@@ -317,11 +319,10 @@ def transmit(self, transport, data, metadata, content_type, timeout=None):
                 ACL list.
             content_type (str): The content type of the resource, e.g. a JPEG
                 image has content type ``image/jpeg``.
-            timeout (Optional[Union[float, Tuple[float, float]]]):
```

Comment:
Again, if this and the deleted blank lines in this file weren't meant to be changed, update them accordingly so that there isn't a diff.